src/components/Question.jsx

```diff
@@ -540,6 +538,14 @@ Question.propTypes = {
   index: PropTypes.number,
   withoutCard: PropTypes.bool,
   collapsible: PropTypes.bool, // Whether the section is collapsible (if the question is a section)
+  options: PropTypes.object,
+  step: PropTypes.object,
+  onNextStep: PropTypes.func,
+  onPreviousStep: PropTypes.func,
+  mapComponent: PropTypes.func,
+  stepIndex: PropTypes.number,
```

Comment:
hmm .... this is a little sad that every question needs to know all this (onNextStep, onPreviousStep ....). Does it make sense for the question to have it? Maybe we should have a talk about it. 

Without looking into it in sufficient detail, to me does not look much like composition instead of inheritance -- this is rather merging functionality into one class.

Moreover, not sure what is difference between index and stepIndex.

Reply:
No, it is my mistake, I should have left it in WizardStep component. I moved it when I was testing and forgot to change it back...